allensdk/test/brain_observatory/sync_utilities/test_sync_stim_get_start_frames.py

```diff
@@ -0,0 +1,279 @@
+import pytest
+import numpy as np
+from allensdk.brain_observatory.sync_dataset import Dataset as SyncDataset
+from allensdk.brain_observatory.sync_stim_aligner import (
+    _get_start_frames,
+    get_start_frames_from_stimulus_blocks)
+
+
+class DummyStim(object):
+    """
+    A class that implements num_frames;
+    used for testing APIs that require
+    _StimulusFile objects.
+    """
+
+    def __init__(self, n_frames):
+        self._n_frames = n_frames
+
+    def num_frames(self):
+        return self._n_frames
+
+
+@pytest.fixture
+def line_name_fixture():
+    return ['lineA', 'stim_running', 'lineB', 'vsync_stim']
+
+
+@pytest.fixture
+def line_to_edges_fixture(
+        sync_sample_fixture):
+    n_samples = len(sync_sample_fixture)
+
+    result = dict()
+
+    # fill lineA and lineB with random bits
+    rng = np.random.default_rng(66123)
+    indexes = np.arange(0, n_samples, 3)
```

Comment:
The three here looks to be also used on line 60. I realize this is in a fixture, but if you are use the same value in multiple places (and they need to be changed together if one changes) you should specify it as a variable.